.github/workflows/test.yml

```
@@ -3,7 +3,30 @@ on:
   pull_request:
 
 jobs:
+  filter_paths:
+    runs-on: ubuntu-20.04
+    outputs:
+      lint: ${{ steps.filter.outputs.lint }}
```

There is not `lint_files` definition.

```suggestion
      lint_files: ${{ steps.filter.outputs.lint_files }}
```

---

Thank you for your comment.
I have corrected it in commit fa70296.